src/main/resources/database/2021-10-05/11-create-reservation.sql

```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+--changeset rmecwaldowski:4
+
+CREATE TABLE reservation(
+	id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    status ENUM (IN_PROGRESS, REALIZED, CANCELED),
+    flight_id BIGINT NOT NULL
+);
+
+ALTER TABLE reservation
+             ADD CONSTRAINT reservation_id
```

Comment:
this FK constraint can be done within CREATE TABLE
are you using right id for FOREIGN KEY (id)?